atomicapp/nulecule/container.py

```diff
@@ -31,10 +31,14 @@ def __init__(self, dryrun=False, docker_cli='/usr/bin/docker'):
             except subprocess.CalledProcessError as e:
                 if "client and server don't have same version" in e.output \
                         or "client is newer than server" in e.output:
-                    print("\nThe docker version in this Atomic App differs "
-                          "greatly from the host version.\nPlease use a different "
-                          "Atomic App version for this host.\n")
-                raise e
+                    raise DockerException("\nThe docker version in this Atomic App "
+                                          "differs greatly from the host version.\n"
+                                          "Please use a different Atomic App version "
+                                          "for this host.\n")
+                elif "Is your docker daemon up and running" in e.output or \
+                     "Are you trying to connect to a TLS-enabled daemon without TLS" \
+                     in e.output:
+                    raise DockerException("Could not connect to the docker daemon.")
```

Comment:
@containscafeine 
yes! that's much better. since Docker may change those sentences / texts in the future and screw up the logging output as well as give the user a bit more information as to why Docker isn't working :)